velox/exec/tests/TableWriteTest.cpp

```diff
@@ -72,14 +73,21 @@ struct TestParam {
       CommitStrategy commitStrategy,
       HiveBucketProperty::Kind bucketKind,
       bool bucketSort,
-      bool multiDrivers) {
-    value = static_cast<uint64_t>(!!multiDrivers) << 40 |
+      bool multiDrivers,
+      CompressionKind compressionKind) {
+    value = static_cast<uint64_t>(compressionKind) << 48 |
+        static_cast<uint64_t>(!!multiDrivers) << 40 |
         static_cast<uint64_t>(fileFormat) << 32 |
         static_cast<uint64_t>(testMode) << 24 |
         static_cast<uint64_t>(commitStrategy) << 16 |
         static_cast<uint64_t>(bucketKind) << 8 | !!bucketSort;
   }
 
+  CompressionKind compressionKind() const {
```

Comment:
I think we can just add a filed in test class with default zstd compression algorithm. And add a dedicated test case with all table layouts that test all the compression kinds which include supported and unsupported. Thanks!